### DISCUSSION TITLE
Keep Focus On Tree After Opening File

### DISCUSSION THREAD

**Discussion by austinliuigi** [nvim-tree/nvim-tree.lua#1591] [Show and tell]
### Is your feature request related to a problem? Please describe.
I personally find it unintuitive for the cursor to jump to a file after opening it in the tree.
<br>
### Describe the solution you'd like
A setting to keep the focus on the tree after opening a file via actions such as `edit`, `vsplit`, `split`, etc.

  *or*

Separate actions similar to `preview`, but for `vsplit`, `split`, etc.
<br>
### Describe alternatives you've considered
I have implemented a simple action_cb to focus on NvimTree after opening a file, but I'm not sure how to implement the window picker that the default actions have.

For example:
```lua
function(node)
  vim.cmd("vsplit" .. node.absolute_path .. " | wincmd L | NvimTreeFocus")
end
```

**Comment by alex-courtis** [comment]
You can use the API to edit the file and then focus the tree. See `:help nvim-tree-api`

```lua
function(node) 
	local nt_api = require("nvim-tree.api")
	nt_api.node.open.edit(node)
	nt_api.tree.focus()
end
```

This isn't necessary as a feature as everything is present in the API.
